apps/cli/main.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+// NOTE: for now the following needs to be in place: ./server/index.cjs, ./client, ./external
+
+const ontimeServer = require('./server/index.cjs');
+const { initAssets, startServer, startIntegrations, shutdown } = ontimeServer;
+
+async function startOntime() {
+  await initAssets();
+
+  await startServer();
+
+  await startIntegrations();
```

Comment:
question: should we provide some feedback here?
This is what we get in electron
```ts
    console.log('Request: Initialise assets...');
    await initAssets();
    console.log('Request: Start server...');
    await startServer();
    console.log('Request: Start integrations...');
    await startIntegrations();
```

Reply:
All logging should go through the logger in the server so I would add these there and go through the logger. No more console.log

Reply:
I will leave this for the broader logging cleanup. I think these should be on the server to log out.